studio-docs/source/contracts.mdx

```diff
@@ -91,27 +91,30 @@ type Query {
   topProducts: [Product!]!
 }
 
-type Product @key(fields: "upc") {
+# All fields on the "Product" object inherit the "partner" tag so we can avoid tagging them explicitly
+type Product @key(fields: "upc") @tag(name: "partner") {
   upc: ID!
   name: String!
   description: String!
-  # Tagged fields
+  # The internalId, and percentageMatch fields would also inherit the "partner" tag, so they need
+  # the "private" and "experimental" tag (and an exclude filter) to keep being inaccessible
   internalId: ID! @tag(name: "private")
   percentageMatch: Float! @tag(name: "experimental")
 }
 ```
 
-This schema applies the `@tag` directive to two object fields: `internalId` and `percentageMatch`.
+This schema applies the `@tag` directive to one object: `Product`, and two object fields: `internalId` and `percentageMatch`. 
```

Comment:
one object (`Product`) and two object fields (`internalId` and `percentageMatch`).